src/packages/cli/src/Generate.ts

```diff
@@ -74,6 +74,10 @@ export class Generate implements Command {
             `✔ Generated ${chalk.bold(name!)}${version ? ` (version: ${version})` : ''
             }${toStr} in ${formatms(after - before)}\n`,
           )
+          const useMessage = await generator.getUseMessage()
```

Comment:
this promise never finishes for generators compiled without generator-helper changes...

Reply:
this is resolved here: GeneratorProcess.ts like 207
```
const timeout = setTimeout(
        () => resolve(''),
        1000,
      )
```